front-end/js/googleMap.js

```diff
@@ -0,0 +1,6 @@
+function initMap() {
+	var map = new google.maps.Map(document.getElementById('map'), {
+		center: {lat: -34.397, lng: 150.644},
```

Comment:
Where does the `center` variable point?

Reply:
Currently, I put a random value. This point is wework.